binding/tfjs_backend.cc

```diff
@@ -59,6 +60,10 @@ TFJSBackend::~TFJSBackend() {
   }
 }
 
+std::unique_ptr<TFJSBackend> TFJSBackend::Create(napi_env env) {
+    return std::unique_ptr<TFJSBackend>(new TFJSBackend(env));
```

Comment:
Make sure to run `yarn format` to auto-fix C++ files.

https://github.com/tensorflow/tfjs-node/blob/master/package.json#L16